src/rust/engine/src/nodes.rs

```diff
@@ -1040,14 +1040,16 @@ impl NodeKey {
   fn user_facing_name(&self) -> Option<String> {
     match self {
       NodeKey::Task(ref task) => task.task.display_info.desc.as_ref().map(|s| s.to_owned()),
-      NodeKey::Snapshot(_) => Some(format!("{}", self)),
+      NodeKey::Snapshot(ref s) => Some(format!("Snapshotting: {}", s.0)),
```

Comment:
`Snapshotting` might not mean much to non-Pants devs. I can't think of a better name though. More something to consider medium term.

Reply:
The content of the `PathGlobs` object should help... it reads as `Snapshotting: PathGlobs(globs=['thing.py'], ..)`

Reply:
Speaking of which, the `PathGlobs` help is very verbose. In a followup, I think it'd be great if it only showed `PathGlobs(['foo', '!ignore'])` and not the other fields.